composeml/label_times/object.py

```diff
@@ -94,56 +113,70 @@ def settings(self):
     @property
     def is_discrete(self):
         """Whether labels are discrete."""
-        return self.label_type == 'discrete'
+        return self.target_types.eq('discrete')
 
     @property
     def distribution(self):
         """Returns label distribution if labels are discrete."""
-        if self.is_discrete:
+        self._assert_single_target()
+        target_column = self.target_columns[0]
+
+        if self.is_discrete[self.target_columns[0]]:
```

Comment:
This can be `if self.is_discrete[target_column]` since the `target_column` variable was created above.